src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy

```diff
@@ -60,78 +64,129 @@ import wooga.gradle.releaseNotesGenerator.utils.ReleaseBodyStrategy
  * Example:
  * <pre>
  * {@code
- *     plugins {
- *         id "net.wooga.release" version "0.15.1"
- *     }
- * }
+ *     plugins {*         id "net.wooga.release" version "0.15.1"
+ *}*}
  * </pre>
  */
 class ReleasePlugin implements Plugin<Project> {
 
     private static Logger logger = Logging.getLogger(ReleasePlugin)
 
-    static final String ARCHIVES_CONFIGURATION = "archives"
-    static final String UNTIY_PACK_TASK = "unityPack"
+    static final String GROUP = "Wooga"
+    static final String EXTENSION_NAME = "atlasRelease"
+
     static final String CLEAN_META_FILES_TASK = "cleanMetaFiles"
     static final String SETUP_TASK = "setup"
     static final String RC_TASK = "rc"
+    static final String FINAL_TASK = "final"
+    static final String SNAPSHOT_TASK = "snapshot"
     static final String TEST_TASK = "test"
-    static final String GROUP = "Wooga"
-    static final String EXTENSION_NAME = "atlasRelease"
+    static final String RELEASE_NOTES_DESCRIPTION_TASK = "releaseNotes"
+    static final String RELEASE_NOTES_MD_TASK = "releaseNotesMD"
```

Comment:
this one to
`RELEASE_NOTES_TASK = "generateReleaseNotes"`

Reply:
What about generateReleaseDescription, generateReleaseNotes? 
Since the original task was for the description of a release, on the page on github for a given release, whereas the notes are committed to the repo. 
I have thought about it since last week and I think its very easy to mix them up if we use the term "notes" on both of them.

Reply:
And since having a description being a string is a common use case.

Reply:
We always need at least two tasks. One to generate the specific release notes (github release body and body of the file) and a task to publish these (github publish vs the new task we have to add).
The generation tasks are only helper tasks. And I would simply name them with the current domain in mind. One is the general release notes the other one github release notes. I don't know what the difference between `RELEASE_NOTES_DESCRIPTION_TASK` vs `RELEASE_NOTES_MD_TASK` is by just looking. Thats why I would add `github`. `release` into the name of the task so one can see that it is specific for github. We could use the term `GithubReleaseBody` here as well. Since we set the body property of the github release with the content of the generated release notes.